backend-go/.golangci.yml

```diff
@@ -0,0 +1,23 @@
+run:
+  deadline: 5m
+
+linters:
+  enable:
+    - errcheck
+    - goimports
+    - govet
+    - goconst
+    - gocyclo
+    - gofmt
+    - revive
+    - goprintffuncname
+    - gosec
+    - lll
+    - misspell
+    - nakedret
+    - wrapcheck
+  disable:
+    - gosimple
+    - staticcheck
+    - structcheck
+    - unused
```

Comment:
Need to add [POSIX newline](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline).